src/OpenTelemetry.Instrumentation.Runtime/README.md

```diff
@@ -60,6 +60,138 @@ using var meterProvider = Sdk.CreateMeterProviderBuilder()
+| process.runtime.dotnet.**jit.compilation_time**       | The amount of time the JIT compiler has spent compiling methods since the process start. The value will be zero under ahead-of-time (AOT) compilation mode.                                                                                                                                                                                  | `ns`        | ObservableCounter | `Int64`    |                  |                  |
+
+The APIs used to retrieve the values are:
+
+* [JitInfo.GetCompiledILBytes](https://docs.microsoft.com/dotnet/api/system.runtime.jitinfo.getcompiledilbytes?view=net-6.0#system-runtime-jitinfo-getcompiledilbytes(system-boolean)):
+  Gets the number of bytes of intermediate language that have been compiled.
+The scope of this value is global. The same applies for other JIT related metrics.
+
+* [JitInfo.GetCompiledMethodCount](https://docs.microsoft.com/dotnet/api/system.runtime.jitinfo.getcompiledmethodcount?view=net-6.0#system-runtime-jitinfo-getcompiledmethodcount(system-boolean)):
+  Gets the number of methods that have been compiled.
+
+* [JitInfo.GetCompilationTime](https://docs.microsoft.com/dotnet/api/system.runtime.jitinfo.getcompilationtime?view=net-6.0#system-runtime-jitinfo-getcompilationtime(system-boolean)):
+  Gets the amount of time the JIT Compiler has spent compiling methods.
+
+### Threading related metrics
+
+These metrics are only available when targeting .NET Core 3.1 or later.
+
+| Name                                                         | Description                                                                                                                                                                                                                                                                         | Units       | Instrument Type   | Value Type | Attribute Key(s) | Attribute Values |
+|--------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------|-------------------|------------|------------------|------------------|
+| process.runtime.dotnet.**monitor.lock_contention.count**     | The number of times there was contention when trying to acquire a monitor lock since the process start. Monitor locks are commonly acquired by using the lock keyword in C#, or by calling Monitor.Enter() and Monitor.TryEnter().                                                  | `{times}`   | ObservableCounter | `Int64`    |                  |                  |
```

Comment:
```suggestion
| process.runtime.dotnet.**monitor.lock_contention.count**     | The number of times there was contention when trying to acquire a monitor lock since the process start. Monitor locks are commonly acquired by using the lock keyword in C#, or by calling Monitor.Enter() and Monitor.TryEnter().                                                  | `{contentions}`   | ObservableCounter | `Int64`    |                  |                  |
```

Reply:
The word "contention" here seem to be an uncountable noun. I don't think using plural form would be good.

1. Meaning no. 1 in [this dictionary](https://www.oxfordlearnersdictionaries.com/us/definition/american_english/contention#:~:text=2%5Bcountable%5D%20contention%20(that,I%20would%20reject%20that%20contention.)
2. The official documentation didn't add "a" in front of the word: "the number of times there was **contention**".
3. The word actually has a second meaning in the dictionary above. And it is countable.

Reply:
The semantic convention [for instrumentation units](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/metrics/semantic_conventions#instrument-units) doesn't have recommendation on what unit to use for **uncountable** nouns.

I think "times" is not perfect but it's okayish. Any suggestion on what to use?

Reply:
I think technically we couldn't say a "count" of an uncountable noun (a water, five water), but we could say a countable unit of the uncountable noun (five bottles of water with "{bottles}" as the unit, or two drops of rain, or a ray of sunlight, or two times of contention with "{times}" as the unit). There might be a better choice than "times" but I hope someone who's better at the English language can help answer it.

Reply:
Updated to `{contended_acquisitions}`. (https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/475#discussion_r915166232)